### DISCUSSION TITLE
How do I style a cell/header in v8?

### DISCUSSION THREAD

**Discussion by morgs32** [TanStack/table#4191] [General]
Anyone know?

**Comment by LinnJS** [comment]
Here is a example of how I styled a basic table using [TailwindCSS](https://tailwindcss.com/)

https://codesandbox.io/s/tanstack-table-react-type-error-reproduction-ii092f?file=/src/Table.tsx

You can also add meta to the `ColumnDef` as described in docs [here](https://tanstack.com/table/v8/docs/api/core/column-def#meta)

Here is a example of its usage from a code sample in another discussion:
https://github.com/TanStack/table/discussions/4184

You can see how they are passing in classNames via meta in their code sample. 

I hope this helps, also in the code standbox examples they have index.css files that target the raw HTML elements. TanStack Table is headless so its intended to have no styles and you have to decide on how to style it.
